chia-blockchain/templates/monit_config.j2

```diff
@@ -10,3 +10,4 @@ check host localhost with address 127.0.0.1
     {% endif %}
 {% endif %}
     if 5 restarts within {{ 5 * chia_monit_failure_threshold }} cycles then timeout
+    if 1 restarts within {{ chia_monit_failure_threshold }} cycles then exec "/etc/monit/monit-notify.sh"
```

Comment:
We should probably wrap this with an option to enable it with it disabled/false in defaults in the role itself

Reply:
This has been wrapped.